tests/phpunit/payment-tokens/class-wc-stripe-payment-tokens-test.php

```
@@ -267,6 +267,121 @@ public function provide_test_get_duplicate_token() {
+		return (object) [
+			'id'                            => 'pm_new_' . $fingerprint,
+			'type'                          => WC_Stripe_Payment_Methods::CARD,
+			WC_Stripe_Payment_Methods::CARD => (object) [
+				'brand'       => $brand,
+				'exp_month'   => $exp_month,
+				'exp_year'    => $exp_year,
+				'last4'       => '4242',
+				'fingerprint' => $fingerprint,
+			],
+		];
+	}
```

PHPMD (2.15.0)

[error] The method make_cc_payment_method is not named in camelCase. (undefined)

(CamelCaseMethodName)